.github/workflows/Benchmarks.yml

```diff
@@ -17,33 +17,12 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        include:
-          - build: windows-2022-release-amd
-            os: [self-hosted, Windows, X64, "1ES.Pool=hld-win2022-amd"]
-            hypervisor: hyperv
-            arch: amd
-          - build: linux-kvm-release-amd
-            os: [self-hosted, Linux, X64, "1ES.Pool=hld-kvm-amd"]
-            hypervisor: kvm
-            arch: amd
-          - build: linux-mshv-release-amd
-            os: [self-hosted, Linux, X64, "1ES.Pool=hld-mshv-amd"]
-            hypervisor: mshv
-            arch: amd
-          - build: windows-2022-release-intel
-            os: [self-hosted, Windows, X64, "1ES.Pool=hld-win2022-intel"]
-            hypervisor: hyperv
-            arch: intel
-          - build: linux-kvm-release-intel
-            os: [self-hosted, Linux, X64, "1ES.Pool=hld-kvm-intel"]
-            hypervisor: kvm
-            arch: intel
-          - build: linux-mshv-release-intel
-            os: [self-hosted, Linux, X64, "1ES.Pool=hld-mshv-intel"]
-            hypervisor: mshv
-            arch: intel
+        hypervisor: [hyperv, mshv, kvm] # hyperv is windows, mshv and kvm are linux
+        cpu: [amd, intel]
+        config: [debug, release]
```

Comment:
Don't we only want to run benchamrks on release?

Reply:
Oh yeah you're right. Removed